ProcessOptimizer/model_systems/model_system.py

```diff
@@ -39,6 +42,7 @@ def __init__(self, score, space, true_min=None):
             scores = [score(point) for point in points]
             true_min = np.min(scores)
         self.true_min = true_min
+        self.set_noise_model(noise_model)
```

Comment:
I'm not really very strong in initiation of classes. 
Would it not be simpler to skip the "set_noise_model" method of the class and then just do this during the initialization:
`self.noise_model = parse_noise_model(noise_model, **kwargs)`?

Reply:
Then we couldn't change the noise model later, which me might want to. But it is a valid question whether we would ever do that.

Reply:
It is there so you can easily change the noise model after initialisation, but that is probably more functionality than we want.  I will remove it.